.github/dependabot.yaml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
that's a pretty high frequency - the default setting now is once a week

Reply:
My idea was to merge those automatically into the `dependabot-updates` all the time and then merge `dependabot-updates` into master once a week. This would give us one PR to master per week instead of once a week several PRs to master. Not sure if that will work, but that's what I wanted to try :)